app/views/playlists/all.html.erb

```diff
@@ -1,24 +1,28 @@
-<div id="static_header">
-  <div class="title">
-    <h1 class="static">Browse all alonetone playlists</h1>
-  </div>
-</div>
 <% if white_theme_enabled? %>
 
   <div class="playlists_container">
-  
-  <ul class="playlists">
-    <%= render partial: 'shared/playlist_white', collection: @playlists, as: :playlist %>
-  </ul>
+    <h2>Browse Playlists</h2>
 
+    <ul class="playlists">
+      <%= render partial: 'shared/playlist_white', collection: @playlists, as: :playlist %>
+    </ul>
   </div>
 
 <% else %>
+
+<div id="static_header">
+  <div class="title">
```

Comment:
Is there a reason this `div` needs to be here, or can we toast it? Seems like a lot of nesting for an h1...

Reply:
I think that's just legacy for the old dark theme at this point. static_header and static_content are two labels that will go away at the dawn of the theme unity!